app/scripts/views/permissions.js

```diff
@@ -0,0 +1,141 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+define([
+  'cocktail',
+  'underscore',
+  'views/form',
+  'views/base',
+  'stache!templates/permissions',
+  'lib/promise',
```

Comment:
Ooohhh!!! I think i saw a grunt task for this the other day: [**grunt-amdcheck**](https://www.npmjs.com/package/grunt-amdcheck).

I did a quick test against **master** with this, **grunttasks/amdcheck.js**:
```js
module.exports = function (grunt) {
  grunt.config('amdcheck', {
    app: {
      files: [{
        expand: true,
        cwd: 'app/',
        src: [
          '**/*.js',
          '!bower_components/**'
        ],
        dest: 'build/'
      }]
    }
  });
}
```

And got a lot of weird output, which may point to some unused and unloved requires:

> Total unused dependencies: 89 in 64 files.
> Total processed files: 251

The output is actually pretty long, so I'll file a separate bug which we can z-never it.